### ISSUE TITLE
Missing resolver information in documentation

### ISSUE THREAD

**Issue by broilogabriel** [sbilinski/sbt-ecr#24]
I only found information about the resolver in your blog post:

`resolvers += Resolver.url("bintray-sbilinski", url("http://dl.bintray.com/sbilinski/maven"))(Resolver.ivyStylePatterns)`

http://mintbeans.com/sbt-amazon-ecr/

I'll create a PR later to update the README to something similar as you've written in your blog post.

**Comment by sbilinski** [comment]
Having the default resolver settings should be enough since #1 was introduced (9729e1f). Did you experience any issues with the base setup in that matter?

**Comment by broilogabriel** [comment]
Yes, I wans't able to download the library without adding the resolver @sbilinski

**Comment by sbilinski** [comment]
Just to clarify - are you using the latest version for the moment (i.e. `0.10.0`) or the one from the blog post?
